2023-06-20 08:29:01 | [INFO] | (main.py).|main(88) | Number of closed cash shifts - 0
2023-06-20 08:29:01 | [WARNING] | (main.py).|main(89) | l_workshift - ()
2023-06-20 08:31:11 | [ERROR] | (request.py).|getQueryShop(62) | HTTPConnectionPool(host='192.168.252.250', port=80): Max retries exceeded with url: /unf_vz_copy/hs/GetChangeShop/V1/GetShop (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f47e78d1570>, 'Connection to 192.168.252.250 timed out. (connect timeout=None)'))
2023-06-20 08:31:11 | [INFO] | (main.py).|main(107) | Change in stores - None
